$NetBSD: patch-xen_arch_x86_boot_build32.lds,v 1.1 2023/11/21 11:38:26 bouyer Exp $
linux's toolchain doesn't generate a .eh_frame section but NetBSD does.
remove it.

--- xen/arch/x86/boot/build32.lds.orig	2023-10-10 16:16:15.693245700 +0200
+++ xen/arch/x86/boot/build32.lds	2023-10-10 16:18:43.427188663 +0200
@@ -31,6 +31,11 @@
         *(.bss.*)
   }
 
+  /* discard .eh_frame section generated by NetBSD toolchain */
+  /DISCARD/ : {
+	*(.eh_frame)
+  }
+
   /* Dynamic linkage sections.  Collected simply so we can check they're empty. */
   .got : {
         *(.got)
